lib/safe-pg-migrations/base.rb

```diff
@@ -70,8 +71,16 @@ def config
   end
 
   module Migration
+    module ClassMethods
+      def safe_pg_migrations_verbose(verbose = nil)
+        @_safe_pg_migrations_verbose = verbose unless verbose.nil?
```

Comment:
Don't know if you tried it, but I would add an attr_reader for this variable, and read it directly in the verbose? method, with something like:
```ruby
def verbose?
  return current_migration.class.safe_pg_migrations_verbose if defined?(current_migration.class.safe_pg_migrations_verbose)
  ...
end
```
So we don't need to pass it around.

Reply:
Good point!